build.gradle

```diff
@@ -77,13 +69,15 @@ apply plugin: 'opensearch.testclusters'
 apply plugin: 'opensearch.pluginzip'
 
 def sqlJarDirectory = "$buildDir/dependencies/opensearch-sql-plugin"
+def jsJarDirectory = "$buildDir/dependencies/opensearch-job-scheduler"
+def adJarDirectory = "$buildDir/dependencies/opensearch-anomaly-detection"
 
 configurations {
     zipArchive
     all {
         resolutionStrategy {
             force "org.mockito:mockito-core:5.8.0"
-            force "com.google.guava:guava:32.1.3-jre" // CVE for 31.1
+            force "com.google.guava:guava:32.1.2-jre" // CVE for 31.1
```

Comment:
Can we add a comment why we are not using the latest version?  Mend will generate an auto-bump for this (if it hasn't already) and would like to include a reason when closing it to prevent future bumps...

Reply:
Thanks for the catch. Reverted this as it's a leftover change from my inter-plugin dependency issues.